### ISSUE TITLE
Implement track focus state keyboard nav

### ISSUE THREAD

**Issue by grliszas14** [audacity/audacity#8891]
### Description

- Pressing 'up' / 'down' moves the selection state between tracks
- Holding 'shift' while holding up/down allows the user to multi-select tracks
- Pressing 'enter' should select/deselect the track
- When 'selecting' a track using our new accessibility system, the blue cursor should highlight the selected track accordingly.

From @Tantacrul:
@DilsonsPickles @LWinterberg  - we'll need to have a discussion about this on our next design call to discuss. Once we're reasonably happy, we'll also need to bring in DB to show him what we've done. I'd suggest a call with him.

[Figma link](https://www.figma.com/design/oQEvoOqeKHQlqabmFmM0Mp/10---Selection?node-id=1169-24820&t=23yBLuEUckO1gQ76-4)

**Comment by DilsonsPickles** [comment]
Added Figma link into task description.

@Tantacrul - I've made the changes we discussed during our call. I'll create a separate ticket to handle the new track selection visualisation we wanted.

@grliszas14 - The spec I've attached details the AU3 behaviour that we need to implement. It also contains one additional behaviour whereby the User can hold shift whilst selecting a new track with the track focus to create a ranged track selection. I.e. selecting all tracks between Mono track 1 and 4.

Please let me know if you have any questions!

**Comment by embarc-gabriel** [comment]
This ticket must also implement what is missing on #8893

**Comment by embarc-gabriel** [comment]
Everything was implemented on the sub tasks
